test/src/androidTest/kotlin/io/realm/ImportTests.kt

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+import io.realm.runtimeapi.RealmModule
+import io.realm.util.TestRealmFieldTypes
+import test.Sample
+import test.link.Child
+import test.link.Parent
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class ImportTests {
```

Comment:
I thin we need ` test that makes sure mixing managed and un-managed while copying works fine

Reply:
I added `importMixedManagedAndUnmanagedHierarchy` having an unmanaged root with a managed child.